src/components/Games/Number/NumberMain.js

```diff
@@ -47,28 +60,43 @@ const Input = styled.input`
 `;
 
 export const NumberMain = () => {
-  const [num, setNum] = useState(0);
+  const [num, setNum] = useState(Math.floor(Math.random() *  (9 - 1 +1)) +1);
```

Comment:
you've got 9 - 1 + 1 you can remove the -1 + 1

Reply:
Actually probs want to replace with this 
`const [num, setNum] = useState(RandomNumberInRange(0,9));` 
as you want the number 0 included too